### ISSUE TITLE
vim.uv used in source.lua not available nvim v0.9.5

### ISSUE THREAD

**Issue by justuswilhelm** [hrsh7th/nvim-cmp#2098]
### FAQ

- [X] I have checked the [FAQ](https://github.com/hrsh7th/nvim-cmp/blob/main/doc/cmp.txt) and it didn't resolve my problem.

### Announcement

- [X] I have checked [Breaking change announcement](https://github.com/hrsh7th/nvim-cmp/issues/231).

### Minimal reproducible full config

Pasting my own config here
```lua
local cmp = require'cmp'
cmp.setup.filetype(
    { "python", "svelte", "typescript" },
    {
        snippet = {
            expand = function(args)
                vim.fn["vsnip#anonymous"](args.body) -- For `vsnip` users.
            end,
        },
        preselect = cmp.PreselectMode.None,
        window = {
            completion = cmp.config.window.bordered(),
            documentation = cmp.config.disable,
        },
        mapping = cmp.mapping.preset.insert({
            ['<C-b>'] = cmp.mapping.scroll_docs(-4),
            ['<C-f>'] = cmp.mapping.scroll_docs(4),
            -- ['<C-Space>'] = cmp.mapping.complete(),
            ['<C-e>'] = cmp.mapping.abort(),
            -- Accept currently selected item. Set `select` to `false` to only confirm explicitly selected items.
            ['<CR>'] = cmp.mapping.confirm({ select = false }),
        }),
        sources = cmp.config.sources(
            {
                { name = 'buffer' },
                { name = 'vsnip' },
                { name = 'nvim_lsp' },
                { name = 'orgmode' },
            },
            {
                { name = 'buffer' },
            }
        ),
    }
)

```

### Description

The newest commit https://github.com/hrsh7th/nvim-cmp/commit/746896cb28f596623b167040993290a08852c908 switches to using nvim.uv.hrtime instead of using os.clock. This causes issues with nvim version < 0.10 as this is not available. (See for example https://github.com/neovim/neovim/issues/27957)

Downgrading to be7bd4c5f860c79da97af3a26d489af50babfd4b fixes the issue.

### Steps to reproduce

1. Edit text in buffer that has lsp / cmp active
2. Error message is shown below

### Expected behavior

Now error message is shown

### Actual behavior

Error message is shown
![image](https://github.com/user-attachments/assets/54beaf02-80ec-4e6c-876c-994a0b4bb7a6)

### Additional context

I use the stock nvim that is part of NixOS 24.05.

```
NVIM v0.9.5
Build type: Release
LuaJIT 2.1.1693350652

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "
/nix/store/939b2xyf52q9w9nkp6ll20y6nqbm5ydl-neovim-unwrapped-0.9.5/share/nvim
"

Run :checkhealth for more info
```

**Comment by Shougo** [comment]
Oh...

nvim-cmp does not define minimal support neovim version.
I think it should be defined.

**Comment by hrsh7th** [comment]
fixed ed31156
